docs/DeveloperGuide.md

```diff
@@ -233,6 +233,12 @@ From the notes in the sequence diagram above, for every attribute in the instrum
 prompt to get user to provide information for that attribute. This is done through a series of methods in
 the `TextUi` class.
 
+### Edit instrument feature
+<img src="images/EditInstrumentSequenceDiagram.png" width="600"/>
```

Comment:
<img width="430" alt="ss1" src="https://user-images.githubusercontent.com/60163283/140517129-76b6f8d6-39e7-4ef6-abb1-60e033807ff9.PNG">
need to include the TextUi.displayEditInstrumentFirst... and getParametersToEdit which then calls filterInvalidParameters. Can rename valid attributes to validAttributes (add a note giving an example of valid attributes for like stock for example).  When returning from filterInvalidParameters say return filteredParameters.